fnet/mockfile

```diff
@@ -0,0 +1 @@
+{"test":"test"}
```

Comment:
maybe use a new testdata/ directory it's more idiomatic and call the file payloadTest1.txt

Reply:
I made .testdata/ as a folder name because `make test` was giving error due to go test ../... comment.